app/controllers/concerns/api_updated_since.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require 'active_support/concern'
+
+module ApiUpdatedSince
+  extend ActiveSupport::Concern
+
+  included do
+    class_eval do
+      allow_params :index, [:since]
+    end
+  end
+
+  def resources_base
+    if updated_since?
+      super.where('updated_at >= ?', updated_since).order('updated_at asc')
```

Comment:
Hmm, wonder what this would do if someone tried to combine it with any other filters or sorts in the query params

Reply:
Fortunately ... feeder does neither `?filter` nor `?sort` right now!